slides/src/main/asciidoc/lecture.adoc

```diff
@@ -343,13 +395,32 @@ Cache[shape=box,label=<<table cellspacing="5" cellborder="0"  border="0">
 ----
 
 
-[.notes]
---
-John: explains how cache works in real Kafka cluster
+== Why it's not working
 
-Explains how TopologyTestDriver works
 
---
+[graphviz,"cache1b.png"]
```

Comment:
I added the summarized intermediate states, so now the "animation" proceeds one record at a time. I think it won't take much more time, and it might be a little clearer.